tsconfig.json

```diff
@@ -15,6 +15,6 @@
 	"exclude": [
 		"node_modules",
 		".vscode-test",
-		"uitest-resources"
+		"test-resources"
```

Comment:
what about the `.test-extensions` ? it ishould be also exluded in tsconfig and part if .gitignore WDYT?

Reply:
@djelinek nice point, but for DAP repo :thinking: 
I have taken a look at [config settings](https://www.typescriptlang.org/tsconfig#exclude):
`Important: exclude only changes which files are included as a result of the include setting`

So in general in LSP we do no need exclude  section because we have:

```
	"include": [
		"src"
	],
```
So nothing from: "node_modules", ".vscode-test", "uitest-resources", "test-resources"  can get into the code base 

On the other hand in [DAP](https://github.com/camel-tooling/camel-dap-client-vscode/blob/main/tsconfig.json) we do not have the include section so - `Specifies an array of filenames or patterns that should be skipped when resolving include` the exclude is useless too :thinking: 

I am not confident that I got it right, but now I think we can just update all configs with:
```
	"include": [
		"src"
	],
```
As on example [here](https://www.typescriptlang.org/tsconfig#include) 

WDYT?

Reply:
So I also compiled with/without `include { src }` and do not see any difference, mb due the project structure

Reply:
@djelinek can we resolve it for now and maybe do the ts config review in the separate jira for all extensions?

Reply:
I agree, but please do not forget to open JIRA issue and link to migration epic to have it tracked

Reply:
[FUSETOOLS2-1997](https://issues.redhat.com/browse/FUSETOOLS2-1997)